src/mbgl/sprite/sprite_atlas.cpp

```diff
@@ -189,11 +189,10 @@ void SpriteAtlas::updateDirty() {
             holder.texture = spriteIterator->second;
             if (holder.texture != nullptr) {
                 copy(holder, imageIterator->first.second);
+                ++imageIterator;
             } else {
-                images.erase(imageIterator);
+                images.erase(imageIterator++);
```

Comment:
This should be prefix increment -- `images.erase(++imageIterator)`. (For anyone who doesn't know: the reason is that `std::map::erase` invalidates any iterators pointing to the erased element. Prefix `++` both returns the _original_ value for passing to `erase`, and increments just before the call. This is both very subtle _and_ a standard C++ idiom in this situation.)